app/src/editor/canvas/components/Status.jsx

```diff
@@ -89,3 +89,9 @@ export default class Status extends React.Component {
         )
     }
 }
+
+export const NotEditableStatus = () => (
+    <div className={styles.status}>
+        Can&#39;t save shared canvas
```

Comment:
i.e. I believe it is technically possible for a shared canvas to be writable

Reply:
@mattatgit Should we change this text since shared canvas can also be writable?

Reply:
IMO it should be something like "This canvas is read-only"

Reply:
@tumppi Not sure, bit lost in the complexities of this one atm, so can't really comment till i understand it more clearly.

Reply:
@mattatgit This is the message shown in the bottom right when a canvas is not editable. Currently it says "Can't save shared canvas", but I'm suggesting that this isn't accurate since the problem isn't that the canvas is shared, the problem is that it's not writable for the current user. Not all shared canvases are not writable.

"Canvas is read-only" sounds good to me

Reply:
@timoxley ok, works for me.

Reply:
Using `Canvas is read-only` now as the message 👍